packages/react-chat/src/views/ChatWidget/index.tsx

```diff
@@ -44,21 +43,19 @@ const ChatWidget: React.FC<ChatWidgetProps> = ({ shadowRoot, chatAPI, ready }) =
     ready
   );
 
-  const side = assistant?.position ?? ChatPosition.RIGHT;
-  const position = { bottom: assistant?.spacing.bottom, [side]: assistant?.spacing.side };
+  const side = assistant.position;
+  const position = { bottom: assistant.spacing.bottom, [side]: assistant.spacing.side };
 
   const isStyleSheetResolved = useResolveAssistantStyleSheet(assistant, shadowRoot);
 
   if (!isStyleSheetResolved) return null;
 
   return (
     <Container withChat={isOpen} isHidden={isHidden} className={theme}>
-      {!!assistant && (
-        <LauncherContainer style={position}>
-          <Proactive side={side} messages={proactiveMessages} />
-          <Launcher onClick={open} image={assistant.launcher} />
-        </LauncherContainer>
-      )}
+      <LauncherContainer style={position}>
```

Comment:
in this place it's a bit unclear to me why the assistant check is dropped (what will be rendered if none is found)

Reply:
because of the typing it's not possible for `assistant` to be `undefined` here